test/mysql.autoupdate.test.js

```diff
@@ -222,6 +220,53 @@ describe('MySQL connector', function() {
     verifyMysqlColumnNameAutoupdate(done);
   });
 
+  it('should update the nullable property of "first_name" to false', function(done) {
+    // update the model "required" property
+    var schema = {
+      name: 'ColRenameTest',
+      options: {
+        idInjection: false,
+        mysql: {
+          schema: 'myapp_test',
+          table: 'col_rename_test',
+        },
+      },
+      properties: {
+        firstName: {
+          type: 'String',
+          required: true,
+          length: 40,
+          mysql: {
+            columnName: 'first_name',
+            dataType: 'varchar',
+            dataLength: 40,
+          },
+        },
+        lastName: {
+          type: 'String',
+          required: false,
+          length: 40,
+        },
+      },
+    };
+
+    ds.createModel(schema.name, schema.properties, schema.options);
+
+    // nullable should be updated to false
+    updateProperty(done);
+  });
+
+  function updateProperty(done) {
```

Comment:
@stelescuraul Yes, please move this function into the test case itself. Again I like your idea to put long logics itself into a function but not worthy of it if only one test case uses it.